shub/settings/auth.py

```diff
@@ -30,8 +30,7 @@
     'social_core.backends.twitter.TwitterOAuth',
     'social_core.backends.facebook.FacebookOAuth2',
     'social_core.backends.github.GithubOAuth2',
-    'guardian.backends.ObjectPermissionBackend',
-    'shub.backends.keyrock.KeyrockOAuth2',
+    'social_core.backends.gitlab.GitLabOAuth2',
```

Comment:
do you know why this is an added line here?

Reply:
Mmmm no Idea, it appears after the merge. It should be removed?